fruit_classifier/predict/predict_utils.py

```diff
@@ -6,6 +6,54 @@
 from skimage.transform import resize
 
 
+def draw_text_with_background(output_image,
+                              text="text",
+                              text_offset_x=10,
+                              text_offset_y=25,
+                              font_scale=.7,
+                              font_thickness=2,
+                              font_color=(0, 0, 0),
+                              font_bg_color=(255, 255, 255),
+                              font=cv2.FONT_HERSHEY_SIMPLEX,
+                              padding=5):
+    """
+    Draws text inside a colored box ontop of an image
+
+    Parameters
+    ----------
+    output_image    : cv2 image -  The image
```

Comment:
*This comment is valid for the whole docstring*

Please use [numpy docstring](https://numpydoc.readthedocs.io/en/latest/format.html)

- Only one space between `name` and `type`, like this `name : type`
- Only use valid types (I guess `cv image` should be `np.array, shape (height, width, channels)`)
- Newline between type and description

```suggestion
    output_image : cv2
        The image
```